### DISCUSSION TITLE
Missing recaptcha type dropdown

### DISCUSSION THREAD

**Discussion by jurriaanr** [solspace/craft-freeform#1221] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**

In the documentation for 3/4/5 it is said:

_Choose a reCAPTCHA version and type from the Captcha Type select dropdown._

However the dropddown is not there

**Steps to reproduce**

1. Go to freeform settings -> captchas
2. Add new recaptcha integration

![image](https://github.com/solspace/craft-freeform/assets/7426848/28066b45-e8bd-4d66-8086-1f89cac01222)

**Expected behavior**

Dropdown is there and visible

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.8.9
- Freeform Version: 5.1.12
- Freeform Edition: free
- Fresh Install or Upgrade: fresh

**Additional context**

**Comment by kjmartens** [comment, marked as answer]
Hi @jurriaanr,

For the Freeform Express (free) edition of Freeform, it currently only supports reCAPTCHA v2 Checkbox, so that is why it doesn't display the **Type** setting. If you require v2 Invisible or v3, you will need to switch to the Lite or Pro edition of Freeform. 🙂

**Comment by jurriaanr** [comment]
Ah ok, that was not clear. Thank you!
